commitizen/cli.py

```diff
@@ -20,7 +20,6 @@
         {"name": "--debug", "action": "store_true", "help": "use debug mode"},
         {
             "name": ["-n", "--name"],
-            "default": "cz_conventional_commits",
```

Comment:
by removing it from here we don't get the documentation when doing `--help` :/

Reply:
I actually hardcode it in the help message.

Reply:
Since `argparse` won't tell us whether it's default or not, it's hard for us to know whether we should use the name in the config because user does not enter the name or user wants to use "cz_conventinoal_commits"

Reply:
@Woile Is there any concern on this solution? Or, do you have idea on how I should to fix this issue? 🙂